cdap-hbase-compat-1.1/src/main/java/co/cask/cdap/data2/transaction/queue/coprocessor/hbase11/HBaseQueueRegionObserver.java

```diff
@@ -344,19 +346,21 @@ public boolean next(List<Cell> results, ScannerContext scannerContext) throws IO
           consumerConfig = getConfigCache(env).getConsumerConfig(currentQueue);
         }
 
-        if (consumerConfig == null) {
-          // no config is present yet, so cannot evict
-          return hasNext;
-        }
-
         invalidTxData = false;
         if (state != null) {
-          long txId = QueueEntryRow.getWritePointer(cell.getRowArray(), currentQueueRowPrefix.length);
+          // TODO: figure out how to get rowKey for Salted Queues
+          byte[] rowKey = ShardedHBaseQueueStrategy.ROW_KEY_CONVERTER.apply(CellUtil.cloneRow(cell));
```

Comment:
If you look up couple lines above, you'll see similar usage when calling `QueueEntryRow.isQueueEntry`.